core/src/main/java/org/sonarsource/sonarlint/core/ConnectedSonarLintEngineImpl.java

```diff
@@ -110,7 +110,7 @@ public AnalysisEngine getAnalysisEngine() {
     return analysisContext.get().analysisEngine;
   }
 
-  public AnalysisContext start() {
+  private AnalysisContext start() {
     setLogging(null);
```

Comment:
Can this be removed now?

Reply:
I am not sure as the start() method is also called from another place (restart()). Anyway setting the log output multiple time is harmless.